include/MQClientException.h

```diff
@@ -60,6 +65,10 @@ class ROCKETMQCLIENT_API MQException : public std::exception {
 
   virtual const char* GetType() const throw() { return m_type.c_str(); }
 
+  int GetLine() const throw(){ return m_line;}
```

Comment:
there will no exception for GetLine and GetFile, so need delete throw() for them

Reply:
Already processed